src/overlays/actors/ovl_En_In/z_en_in.c

```diff
@@ -1,15 +1,38 @@
+/*
+ * File z_en_in.c
+ * Overlay: ovl_en_in
+ * Description: Ingo/Gorman Bros
+ */
+
 #include "z_en_in.h"
 
 #define FLAGS 0x00000019
 
 #define THIS ((EnIn*)thisx)
 
+#define MACRO1                                                                                                    \
+    {                                                                                                             \
+        gSaveContext.weekEventReg[92] &= (u8)~7;                                                                  \
+        gSaveContext.weekEventReg[92] = gSaveContext.weekEventReg[92] | (u8)(gSaveContext.weekEventReg[92] & ~7); \
+    }
+
+#define MACRO2                                                                              \
+    {                                                                                       \
+        gSaveContext.weekEventReg[92] &= (u8)~7;                                            \
+        gSaveContext.weekEventReg[92] =                                                     \
+            gSaveContext.weekEventReg[92] | (u8)((gSaveContext.weekEventReg[92] & ~7) | 1); \
+    }
```

Comment:
Even if those patterns repeat all over the place, I think hiding those behind a nameless macro is a bad idea.
Either properly document what those macros are doing or remove them.

Reply:
I used macros not because they repeated, but IDO same line memes, i.e., the commands have to be on the same line to OK. This suggests they were macros. I can work on documenting them though.

Reply:
Decided to go with `SET_FLAGS_FINISH_RACE` and `SET_FLAGS_START_RACE`